### PR TITLE
Updates for trifecta tech ownership and sponsoring

### PR DESCRIPTION
**Pull request by erikjee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,9 +65,9 @@ If you'd like to improve teach-rs, either by doing touchups, restructuring a mod
...
 
 
-The initial sponsor of the project is the Faculty of Informatics and Information Technologies (FIIT) of the Slovak University of Technology (STU) in Bratislava, Slovakia. [FIIT](https://www.fiit.stuba.sk/en.html?page_id=749)'s contribution has enabled us to layout the groundwork for the course. [Tweede golf](https://tweedegolf.nl/en) and [Rust Edu](https://rust-edu.org/) have also contributed substantially to the creation of teach-rs.
+The project's initial sponsor is the Faculty of Informatics and Information Technologies (FIIT) of the Slovak University of Technology (STU) in Bratislava, Slovakia. [FIIT](https://www.fiit.stuba.sk/en.html?page_id=749)'s contribution has enabled us to lay the groundwork for the course. [Tweede golf](https://tweedegolf.nl/en) and [Rust Edu](https://rust-edu.org/) have also contributed substantially to the creation of teach-rs.
 
-In addition, lead developer Henk has received a grant from the [Rust Foundation](https://foundation.rust-lang.org/).
+In addition, one of our maintainers, https://github.com/hdoordt, received a grant from the [Rust Foundation](https://foundation.rust-lang.org/).
```
Comment (hdoordt) [inline README.md:70]:
```suggestion
In addition, one of our maintainers, [@henk](https://github.com/hdoordt), received a grant from the [Rust Foundation](https://foundation.rust-lang.org/).
```

**Comment by erikjee** [comment]
Note: blocked by approval for github sponsoring / stripe

**Comment by erikjee** [comment]
@hdoordt FYI, I'm still waiting for the GitHub sponsoring / stripe approval. For some reason, it is taking unusually long. I'll comment here again when it is ready.

**Comment by erikjee** [comment]
@hdoordt GitHub sponsoring / stripe approval finally came through. Can you merge this now?

**Review by hdoordt — APPROVED** [review]

**Pull request merged**